src/planner/ompl/CRRT.cpp

```diff
@@ -1,68 +1,91 @@
-#include <aikido/planner/ompl/CRRT.hpp>
-#include <aikido/planner/ompl/GeometricStateSpace.hpp>
+#include <limits>
 #include <ompl/base/goals/GoalSampleableRegion.h>
 #include <ompl/tools/config/SelfConfig.h>
-#include <limits>
+#include <aikido/planner/ompl/CRRT.hpp>
+#include <aikido/planner/ompl/GeometricStateSpace.hpp>
 
 namespace aikido {
 namespace planner {
 namespace ompl {
 //=============================================================================
-CRRT::CRRT(const ::ompl::base::SpaceInformationPtr &_si) : CRRT(_si, "CRRT") {}
+CRRT::CRRT(const ::ompl::base::SpaceInformationPtr& _si) : CRRT(_si, "CRRT")
+{
+}
 
 //=============================================================================
-CRRT::CRRT(const ::ompl::base::SpaceInformationPtr &_si,
-           const std::string &name)
-    : ::ompl::base::Planner(_si, name), mGoalBias(0.05), mMaxDistance(0.1), 
-    mLastGoalMotion(nullptr), mCons(nullptr), mMaxStepsize(0.1), mMinStepsize(1e-4) {
-
-  auto ss =
-      ompl_dynamic_pointer_cast<GeometricStateSpace>(si_->getStateSpace());
-  if (!ss) {
+CRRT::CRRT(
+    const ::ompl::base::SpaceInformationPtr& _si, const std::string& name)
+  : ::ompl::base::Planner(_si, name)
+  , mGoalBias(0.05)
+  , mMaxDistance(0.1)
+  , mLastGoalMotion(nullptr)
+  , mCons(nullptr)
+  , mMaxStepsize(0.1)
+  , mMinStepsize(1e-4)
+{
+
+  auto ss
+      = ompl_dynamic_pointer_cast<GeometricStateSpace>(si_->getStateSpace());
+  if (!ss)
+  {
     throw std::invalid_argument(
         "CRRT algorithm requires a GeometricStateSpace");
   }
 
   specs_.approximateSolutions = true;
   specs_.directed = true;
 
-  Planner::declareParam<double>("range", this, &CRRT::setRange, &CRRT::getRange,
-                                "0.:1.:10000.");
-  Planner::declareParam<double>("goal_bias", this, &CRRT::setGoalBias,
-                                &CRRT::getGoalBias, "0.:.05:1.");
-  Planner::declareParam<double>("proj_res", this, &CRRT::setProjectionResolution,
-                                &CRRT::getProjectionResolution, "0.:1.:10000.");
-Planner::declareParam<double>("min_step", this, &CRRT::setMinStateDifference,
-                                &CRRT::getMinStateDifference, "0.:1.:10000.");
-  
+  Planner::declareParam<double>(
+      "range", this, &CRRT::setRange, &CRRT::getRange, "0.:1.:10000.");
+  Planner::declareParam<double>(
+      "goal_bias", this, &CRRT::setGoalBias, &CRRT::getGoalBias, "0.:.05:1.");
+  Planner::declareParam<double>(
+      "proj_res",
```

Comment:
Nit: Unrelated to this PR, rename this to `projection_resolution`. We're not charged by the byte. 😬